frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -414,10 +403,7 @@ export default ({
       const sendMessage = (beforePrePublish) => {
         let pendingMessageHash = null
         const beforeRequest = (message, oldMessage) => {
-          if (!this.checkEventSourceConsistency(contractID)) return
           sbp('okTurtles.eventQueue/queueEvent', CHATROOM_EVENTS, async () => {
-            if (!this.checkEventSourceConsistency(contractID)) return
-
```

Comment:
Are you sure that these two checks should be removed? If we've switched, we'll try to do things that are unnecessary.